### PR TITLE
fix: all your webpack problems

### PR DESCRIPTION
**Pull request by gdi2290**
yo, I fixed all your webpack problems

### CODE REVIEW

**Comment by robwormald** [comment]
so noParse just prevents webpack from attempting to parse the referenced files, yeah?

**Comment by gdi2290** [comment]
ya

**Comment by gdi2290** [comment]
I added noParse to all the bundled files in the angular2 module

File: webpack.config.js [hunk 1]
```diff
@@ -22,17 +13,18 @@ module.exports = {
...
     loaders: [
-      {
-        test: /\.ts$/,
-        loader: 'ts-loader'
-      },
-      
-    ]
+      { test: /\.ts$/, loader: 'ts-loader' },
+    ],
+    noParse: [ /angular2\/bundles\/.+/ ]
```
Comment (gdi2290) [inline webpack.config.js]: can has noParse bundles

**Comment by robwormald** [comment]
SGTM.

**Comment by robwormald** [comment]
also LGTM

**Comment by robwormald** [comment]
mergerated

**Pull request merged**